src/Microsoft.VisualStudio.Threading/AsyncReaderWriterLock.cs

```diff
@@ -1529,8 +1529,6 @@ private async Task InvokeBeforeWriteLockReleaseHandlersAsync()
                     }
                 }
 
-                await releaser.ReleaseAsync().ConfigureAwait(false);
```

Comment:
❔ Will this still be effectively `ConfigureAwait(false)` in all cases?

Reply:
Yes, because of the `.ConfigureAwait(false)` I added to the using expression.

But FWIW, users don't usually need to go to that trouble. With JTF aware code (which all uses of ARWL in VS are) you don't typically have to be super-careful to put `.ConfigureAwait(false)` everywhere. I only did it here to ensure equivalent behavior to before in case it was significant for some reason I couldn't tell.

Reply:
Ah, misread the parentheses nesting